src/cms/init-cms.ts

```diff
@@ -74,6 +75,7 @@ const cmsConfig: CmsConfig = {
 // (window as any).CMS_CONFIG = cmsConfig;
 
 export function initCMS() {
+  CMS.registerWidget("clue", ClueControl);
   CMS.registerWidget("json", JsonControl);
```

Comment:
Should we remove this since it's no longer used, or leave it in case it's useful to play around with it in the future?